app/index.js

```diff
@@ -148,10 +155,14 @@ app.on('ready', () =>
 
         const hwin = new Window({width, height, x: startX, y: startY}, cfg, fn);
         windowSet.add(hwin);
+        if (mainWindowState.maximized) {
```

Comment:
because here is the only place you read something from it?

Reply:
hwin on (close) is the place where it reads the data of the current window on line 165.
Then it reads the data when next time the app is launched!

Reply:
the one u mentioned just checks the maximized state of the window and sets the window state to either maximized or restored.

Reply:
usage of this small package overrides the default width and height parameters which were being set earlier.